scripts/build/bootstrap/incremental_build_util.py

```diff
@@ -234,29 +234,29 @@ def create_volume(ec2_client, availability_zone, repository_name, project, pipel
         time.sleep(1)
         response = ec2_client.describe_volumes(VolumeIds=[volume_id, ])
 
-    print("Volume {} created\n\tSnapshot: {}\n\tRepository {}\n\tProject {}\n\tPipeline {}\n\tBranch {}\n\tPlatform: {}\n\tBuild type: {}"
-        .format(volume_id, snapshot_id, repository_name, project, pipeline, branch, platform, build_type))
+    print(("Volume {} created\n\tSnapshot: {}\n\tRepository {}\n\tProject {}\n\tPipeline {}\n\tBranch {}\n\tPlatform: {}\n\tBuild type: {}"
+        .format(volume_id, snapshot_id, repository_name, project, pipeline, branch, platform, build_type)))
```

Comment:
I think an extra pair of parentheses was added here. But shouldn't affect the actual script.

Reply:
Yeah, I have another task to cleanup incremental build script, will remove the extra parentheses in another PR.